source/code-rule.rst

```diff
@@ -25,9 +37,25 @@ Code standard
   - Loose-coupling between modules so that the failure of one module can
     have minimal effects on other modules.
 
+  - For 3rd party Python library, implementing either a back up plan or
+    an error/exception handler to avoid breaking the import chain
```

Comment:
```suggestion
  - Third party Python library imports need either back-up implementations or error/exception handling to avoid breaking the import chain
```